crates/rover-client/Cargo.toml

```diff
@@ -22,14 +22,23 @@ backoff = "0.3"
 camino = "1"
 chrono = { version = "0.4", features = ["serde"] }
 git-url-parse = "0.4.0"
-git2 = { version = "0.13", default-features = false, features = ["vendored-openssl"] }
+git2 = { version = "0.13", default-features = false, features = [
+    "vendored-openssl",
+] }
 graphql_client = "0.10"
 http = "0.2"
 humantime = "2.1.0"
 prettytable-rs = "0.8.0"
-reqwest = { version = "0.11", default-features = false, features = ["blocking", "brotli", "gzip", "json", "native-tls-vendored", "socks"] }
+reqwest = { version = "0.11", default-features = false, features = [
```

Comment:
are you using a specific TOML formatter? wonder if we should add one to our linting pipeline

Reply:
I have `Even Better TOML` - which one do you have again? I'll switch to the same one. (I think we've had this conversation a few of my OSs ago 😂)

Reply:
i have that same one installed! maybe i'm just not autoformatting by default or something. meh.